### PR TITLE
Support getPreviewer()

### PR DESCRIPTION
**Pull request by matsui54**
I will fix the import path and add doc if Shougo/ddu.vim#16 is marged.

### CODE REVIEW

File: denops/@ddu-ui-ff/preview.ts [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  ActionFlags,
+  BufferPreviewer,
+  Context,
+  DduItem,
+  DduOptions,
+  NoFilePreviewer,
+  PreviewContext,
+  Previewer,
+  TermPreviewer,
+} from "../../../ddu.vim/denops/ddu/types.ts";
```
Comment (Shougo) [inline denops/@ddu-ui-ff/preview.ts:11]: You cannot use relative path.

**Review by Shougo — CHANGES_REQUESTED** [review]
You need to fix relative path problem.

**Comment by Shougo** [comment]
`ddu#ui#ff#_preview_file()` seems not used.  It should be removed.

File: denops/@ddu-ui-ff/preview.ts (hunk 1 above)
Comment (matsui54) [reply denops/@ddu-ui-ff/preview.ts:11]: Yes, it is for temporary use. So, can you update the ddu version to 1.6.0?

**Comment by Shougo** [comment]
OK.  I have released 1.6.0.

**Comment by matsui54** [comment]
Thanks. It is ready.

**Pull request merged**